axiom/datasets.go

```diff
@@ -453,6 +473,31 @@ func (s *DatasetsService) Query(ctx context.Context, id string, q query.Query, o
 	return &res, nil
 }
 
+// APLQuery executes the given query specified using the Axiom Processing
+// Language (APL).
+func (s *DatasetsService) APLQuery(ctx context.Context, rawQuery string, opts APLQueryOptions) (*query.Result, error) {
```

Comment:
Can we at least have a consistent handling of `{start,end}Time`? Either put it in the options everywhere or nowhere pls.

Reply:
It is two very different languages. But fine with that. Please approve this one and I'll adhere in `axiom-node`.